### DISCUSSION TITLE
Missing Hamster Kombat Airdrop Tokens in EBI Account

### DISCUSSION THREAD

**Discussion by believerbl** [blockscout/blockscout#10837] [Support]
Date:  September 28, 2024

To: EBI Exchange Support Team

I am writing to report an issue with the recent Hamster Kombat (HMSTR) airdrop. According to the Hamster Kombat platform, my airdrop tokens have been successfully claimed and transferred to my EBI account. However, upon checking my EBI account, I noticed that the tokens are not reflected in my balance.

Details:

Airdrop Source: Hamster Kombat
Claim Date: September 26, 2024
Expected Token Amount: 391.400228809

Steps Taken:

1. Verified the claim status on the Hamster Kombat platform, which shows the tokens as successfully claimed.
2. Logged into my EBI account to check the balance, but the tokens are missing.
3. Double-checked the wallet address and transaction details to ensure accuracy.

Request:

I kindly request anyones assistance in resolving this issue. Please investigate about it . If there are any additional steps I need to take, please let me know.

Attachments:

- Screenshot of the claim confirmation from Hamster Kombat

![Screenshot_2024-09-28-01-03-17-70_948cd9899890cbd5c2798760b2b95377.jpg](https://github.com/user-attachments/assets/66f94ee3-37bd-4144-83c8-9faa2e0cf0b5)

- Screenshot of my EBI account balance

![Screenshot_2024-09-28-01-20-56-03_948cd9899890cbd5c2798760b2b95377.jpg](https://github.com/user-attachments/assets/e9e578f0-6f8d-4b6a-9f10-66c324822f5e)

![Screenshot_2024-09-28-01-20-28-71_948cd9899890cbd5c2798760b2b95377.jpg](https://github.com/user-attachments/assets/1350bbe3-44c7-4181-8a42-9bd8b7824173)

Thank you for your prompt attention to this matter. I look forward to your response.

**Comment by Amir1350-nik** [comment]
I also have the same problem with Ebi exchange about my Hamester coin

**Comment by ulyanas** [comment]
Blockscout is an open-source blockchain explorer and is not associated with Ebi Exchange or Ebi Chain. Please direct any reports to their respective support channels.
